packages/field-plugin/src/createFieldPlugin/createPluginActions/createPluginActions.ts

```diff
@@ -103,6 +104,14 @@ export const createPluginActions: CreatePluginActions = (
           // eslint-disable-next-line @typescript-eslint/ban-ts-comment
           // @ts-ignore
           typeof action === 'function' ? action(state.content) : action
+
+        if (!isSerializable(action)) {
```

Comment:
Hey @demetriusfeijoo you mixed two different topics in this PR.

- update doc link in the sandbox
- error message for unserializable data

Please split them into two :)